frontend/packages/kubevirt-plugin/package.json

```diff
@@ -7,7 +7,8 @@
     "@console/internal": "0.0.0-fixed",
     "@console/plugin-sdk": "0.0.0-fixed",
     "@console/shared": "0.0.0-fixed",
-    "kubevirt-web-ui-components": "~0.1.35"
+    "kubevirt-web-ui-components": "~0.1.35",
+    "kubevirt-typescript-api": "^0.18.1-rc"
```

Comment:
@suomiy , can you please provide reasoning behind this new project/dependency?

Reply:
as said in #1731, this is to add TS definitions of kubevirt objects

Reply:
Could we do it without adding this dependency and maybe revisit it later ? It will just slow down merging the PRs and we dont really have a time to waste.